SPECIFICATION.md

```diff
@@ -350,17 +350,17 @@ Lets assume we receive a feed id through an external channel (such as text messa
 
 ## Block prioritization
 
-When deciding which block to download from a remote peer a couple of things should be taken into suggestion.
+When deciding which block to download from a remote peer, a couple of things should be taken into account.
 
-To allow for features such as live booting of linux containers, real-time video and audio streaming, and any many more Hyperdrive allows for higher prioritization of specific ranges of blocks to download.
+To allow for features like live booting linux containers, real-time video and audio streaming, etc... Hyperdrive allows for higher prioritization of specific ranges of blocks to download first.
 
-If no blocks ranges are prioritized a strategy such as "Rarest first" or "Random first piece" should be used similar to the strategy used in BitTorrent. This allows blocks to be spread out evenly across peers without having single peers that are the only uploaders of a specific piece.
+If no blocks ranges are prioritized, a strategy such as "Rarest first" or "Random first piece" should be used similar to the strategy used in BitTorrent. This allows blocks to be spread out evenly across peers without having single peers that are the only uploaders of a specific piece.
```

Comment:
Are there any URLs we can reference that describe "Rarest first" or "Random first piece" that you recommend?